x/cheqd/types/keys.go

```diff
@@ -25,6 +25,7 @@ func KeyPrefix(p string) []byte {
 const (
 	DidKey      = "did:"
 	DidCountKey = "did-count:"
+	// DidNamespaceKey FIXME: Should be `did-namespace:`.
+	// Networks was started with `testnet` value so we need a migration now.
+	DidNamespaceKey = "testnet"
```

Comment:
This looks like a regression?

Reply:
No, a wrong constant was used to take this value, I fixed that and made the fact that we used the wrong value more explicit